packages/tldraw/src/lib/shapes/shared/TextLabel.tsx

```diff
@@ -49,16 +46,18 @@ export const TextLabel = React.memo(function TextLabel({
 	align,
 	verticalAlign,
 	wrap,
-	bounds,
```

Comment:
Also removing bounds here because it's not used (cloud shape uses it but it has no detectable effect).